packages/use-i18n/README.md

```diff
@@ -209,6 +209,38 @@ const App = () => {
 }
 ```
 
+### formatList
+
+This hook exposes a `formatList` function which can be used to format lists of strings.
+
+The first parameter is an array of strings to format.
+
+It accepts an `options` as second parameter which is an [Intl.ListFormat `options` object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat/ListFormat).
+
+```js
+import { useI18n } from '@scaleway/use-i18n'
+
+const App = () => {
+  const { formatList } = useI18n()
+
+  const cities = formatList(['Paris', 'New York', 'London'])
+  // Paris, New Yo
```

Comment:
https://www.youtube.com/watch?v=oMX1sc3eOTE

Reply:
Forgot to remove an old code example 😅